### DISCUSSION TITLE
Is the review for me or a team I'm on?

### DISCUSSION THREAD

**Discussion by CdrMarks** [community/community#8899] [Pull Requests]
I am receiving emailed notification about a requested review because my team is listed in the CODEOWNERS file.   I'm on a team of 11 people and I'm having trouble finding the PRs that need my attention.

# Suggestion: Update the email footer text
The footer of the email I'm receiving has the following text:
> You are receiving this because your review was requested.

If a user wasn't explicitly included as one of the PR's reviewers, I suggest this text be changed to:
> You are receiving this because your **team's** review was requested.

# Am I missing something?
If there is another way to filter emailed to identify individual versus team review requests, I'm happy to give that a try.

**Comment by jsoref** [comment]
Indeed, it isn't just in the email notification, it's also in the PR itself:
<img width="988" alt="image" src="https://user-images.githubusercontent.com/2119212/155588079-ebb2ef6b-87ee-41a2-8ed3-fd792035d314.png">
<img width="255" alt="image" src="https://user-images.githubusercontent.com/2119212/155588101-4915506b-99c0-4ddd-a638-e56582f2433f.png">

The email should say:
> You are receiving this because the x/y team's review was requested, and you're a member.

Bonus points there is a link to the team membership.

The PR view should say
> requested a review from the x/y team (you're a member) on this pull request.

In the example I have, that'd make it:

> [Vlaaaaaaad ](https://github.com/Vlaaaaaaad)requested a review from the [reviewdog](https://github.com/reviewdog) / [actions-maintainer](https://github.com/orgs/reviewdog/teams/actions-maintainer) team (you're a member) on this pull request.

**Reply by jsoref** [reply]
While someone is touching this ui, if they could fix the link to the requester to not include the trailing space, that'd be great...

<img width="311" alt="image" src="https://user-images.githubusercontent.com/2119212/155588878-78f2c037-1b3d-4409-90d0-e8818154c5a4.png">

```diff
-[Vlaaaaaaad ](https://github.com/Vlaaaaaaad)requested your review on this pull request.
+[Vlaaaaaaad](https://github.com/Vlaaaaaaad) requested your review on this pull request.
```

**Reply by jsoref** [reply]
As an aside, there are lots of places where trailing spaces are included in links. Here's another:

<img width="188" alt="image" src="https://user-images.githubusercontent.com/2119212/155589129-854e5f22-5113-4be4-a02c-dd5916e7953a.png">

(It's actually an accessibility fail in addition to everything else.)

-- I suppose at some point I can file a general item on this. But...
